scripts/filters/readingtime.js

```diff
@@ -0,0 +1,17 @@
+var htmlToText = require('html-to-text');
+var wordCount = require('word-count');
+
+hexo.extend.filter.register('after_post_render', function(page) {
```

Comment:
1. `function(page)` use `post` instead of `page`

2. Before calculating, check if a reading time is already set.
``` js
if (post.readingtime) {
  return
}
```